si5351/registers.go

```diff
@@ -0,0 +1,64 @@
+package si5351
+
+// The I2C address which this device listens to.
+const AddressDefault = 0x60     // Assumes ADDR pin is low
+const AddressAlternative = 0x61 // Assumes ADDR pin is high
+
+const (
+	OUTPUT_ENABLE_CONTROL = 3
+
+	CLK0_CONTROL = 16
+	CLK1_CONTROL = 17
+	CLK2_CONTROL = 18
+	CLK3_CONTROL = 19
+	CLK4_CONTROL = 20
+	CLK5_CONTROL = 21
+	CLK6_CONTROL = 22
+	CLK7_CONTROL = 23
```

Comment:
```suggestion
	CLK0_CONTROL = iota + 15
	CLK1_CONTROL
	CLK2_CONTROL
	CLK3_CONTROL
	CLK4_CONTROL
	CLK5_CONTROL
	CLK6_CONTROL
	CLK7_CONTROL
```
This could also be re-written using iota

Reply:
Should't that be iota + 16 ?

Reply:
Yes, you are 100% right, that was a typo, sorry!